api/cache/names.go

```diff
@@ -32,23 +32,18 @@ func NewObjectsNameCache(config *Config) *ObjectsNameCache {
 	return &ObjectsNameCache{cache: gc}
 }
 
-// Get returns a cached object.
-func (o *ObjectsNameCache) Get(key string) *address.Address {
+// Get returns a cached object. Indicates value presence by second return value.
+func (o *ObjectsNameCache) Get(key string) (v oid.Address, ok bool) {
```

Comment:
I guess it is possible to keep signature the same and return `&result` value. All  `Get*` functions in caches return `nil` in case of cache miss, so maybe it's worth to keep it.

Reply:
Tried to get rid of using pointers where it's not mutation-needed. Reverted.